server.go

```diff
@@ -484,6 +491,7 @@ func main() {
 	router.HandleFunc("/api/user", CreateUser).Methods("PUT")
 	router.HandleFunc("/api/user", use(GetUser, basicAuth)).Methods("GET")
 	router.HandleFunc("/api/user", use(UpdateUser, basicAuth)).Methods("POST")
+	// Comment
```

Comment:
Shouldn't be `PUT` for `UPDATE` and `POST` for `CREATE`?

Reply:
YES! You're absolutely right (for instance here: https://restfulapi.net/rest-put-vs-post/ ) . My RestAPI as I defined it is non-standard :) Sorry for this, on my current project (from some historical reasons I can not explain) we are using this non-standard notation and I used it there as well - this should be definitely considered as bug.